### DISCUSSION TITLE
是否准备开放服务器端代码？

### DISCUSSION THREAD

**Discussion by ningpengtao-coder** [opentiny/tiny-engine#132] [Q&A] [answered]
目前ai生成比较有意思，是否有计划开放这部分代码？

**Comment by chilingling** [comment, marked as answer]
服务端代码目前计划在 23.,12 进行开源，但是不包括 ai 部分代码

**Reply by Thea1211** [reply]
请问服务端代码有具体的时间了吗

**Comment by cuivip** [comment]
服务端代码是否已经开放？我只看到了mock的代码

**Comment by chilingling** [comment]
服务端代码已开源，具体代码仓库：

- https://github.com/opentiny/tiny-engine-webservice
- https://github.com/opentiny/tiny-engine-data-center

欢迎提 ISSUE、PR，以及 STAR
@cuivip 
@Thea1211
